### PR TITLE
Urena code/week7 3

### LINKED ISSUE #90: fix order by to handle nums instead of strings (like Index)
done fixes for #5.

Reviewed by @letgotopal

### LINKED ISSUE #93: add the outputTableToDisk to CRUD operations
addresses issue #5 reviewed by @letgotopal

### LINKED ISSUE #95: take care of unsigned int and in comparison warnings
Fixes Bug
No user story, just convenience
reviewed by @letgotopal

### LINKED ISSUE #118: Add a folder just to hold the output of the data
fixes #5 reviewed by @letgotopal

### PR DESCRIPTION
**Pull request by urena-code**
fixes #93 
fixes #118
fixes #95 
fixes #90 
fixes the delete column to properly delete - little bug found
currently add_row still needs to be overwritten - I'm not sure why right now it is commented out

### CODE REVIEW

File: datastruct/DataView.cpp
Comment (urena-code) [inline datastruct/DataView.cpp:1]: general unsigned commparisons,
fixes #90 
small error message - error

File: datastruct/DataView.h
Comment (urena-code) [inline datastruct/DataView.h:1]: not sure why it was a private function

File: datastruct/database.cpp
Comment (urena-code) [inline datastruct/database.cpp:1]: needs to be typecasted to unsigned

File: datastruct/schema.cpp
Comment (urena-code) [inline datastruct/schema.cpp:1]: general unsigned stuff

File: datastruct/schema.h
Comment (urena-code) [inline datastruct/schema.h:1]: must be virtual - this is throwing database warnings - no errors tho

File: datastruct/table.cpp
Comment (urena-code) [inline datastruct/table.cpp:1]: fixes unsigned
adds outputDisk to save data

File: datastruct/table.h
Comment (urena-code) [inline datastruct/table.h:1]: overload the parent functions

**Review by letgotopal — APPROVED** [review]
All changes look good

**Pull request merged**